metals/src/main/scala/scala/meta/internal/metals/DoctorResults.scala

```diff
@@ -56,6 +56,7 @@ final case class DoctorTargetInfo(
   def toJson: Obj =
     ujson.Obj(
       "buildTarget" -> name,
+      "compilationStatus" -> compilationStatus.explanation,
```

Comment:
Shouldn't we bump the version?

Reply:
Ahhh good catch, I totally forgot about the version. Yes I'll bump this and update the docs.